extensions/storages/source/providers/Temporary.ts

```diff
@@ -1,14 +1,14 @@
 import { tmpdir } from 'node:os'
 import { join } from 'node:path'
-import assert from 'node:assert'
-import { pathToFileURL } from 'node:url'
 
 import { FileSystem } from './FileSystem'
 
-export class Temporary extends FileSystem {
-  public constructor (url: URL) {
-    assert.equal(url.protocol, 'tmp:', `Invalid temporary URL: ${url.toString()}`)
+export interface TemporaryOptions {
+  prefix?: string
```

Comment:
Documentation says it should be `path`.

Reply:
Missed that, thank you! Will adjust the docs

Reply:
Fixed in https://github.com/toa-io/toa/pull/493/commits/8df71c999a30a660c2dca97fd31d42711656d0d5